src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -131,4 +165,69 @@ protected void end() {
             }
         };
     }
+
+    public Command DriveDistance(double inches) {
```

Comment:
I think we need to rename this function... "DriveDistance" was supposed to me "I want you to drive for XYZ inches", but now it seems like it means "I want you to drive until you are XYZ incheas AWAY from something"... I'd call it something like DriveToProximity or something